device/edison.go

```diff
@@ -62,19 +62,23 @@ func (d *edison) Configure() error {
 	c.AddConfigFn(config.Interface, config.NewCallbackFn(setupInterface, nil))
 	c.AddConfigFn("xIotit", config.NewCallbackFn(setupIotit, nil))
 
-	if err := d.getIPAddress(); err != nil {
-		log.Error(err)
+	if len(d.IP) == 0 {
+		if err := d.getIPAddress(); err != nil {
+			log.Error(err)
+		}
 	}
 
-	if d.ip == "" {
+	if d.IP == "" {
 		fmt.Println("[-] Can't configure board without knowing it's IP")
 		return nil
+	} else {
```

Comment:
__[golint]__
* [golint] if block ends with a return statement, so drop this else and outdent its block

<!-- Comment by [SideCI](https://sideci.com) -->